server/index.js

```diff
@@ -117,6 +117,30 @@ app.post('/api/products', (req, res, next) => {
   }
 });
 
+app.get('/api/products/cart/:cartId', (req, res, next) => {
+  const token = req.get('X-Access-Token');
+  const payload = jwt.verify(token, process.env.TOKEN_SECRET);
+  const cartId = payload.cartId;
```

Comment:
A couple points here

- since we are using the cartId from the token, we can leave `:cartId` out of the path pattern.
- in the case of having no token, it would be wise to `res.json([])` and `return` in a guard, to avoid querying the database.